x/interchainqueries/keeper/msg_server.go

```diff
@@ -43,11 +43,6 @@ func (k msgServer) RegisterInterchainQuery(goCtx context.Context, msg *types.Msg
 		return nil, sdkerrors.Wrapf(types.ErrNotContract, "%s is not a contract address", msg.Sender)
 	}
 
-	if err := msg.ValidateBasic(); err != nil {
```

Comment:
why have you removed  this one validate, but left in `RemoveInterchainQuery` `UpdateInterchainQuery` methods